runtime/rastrace/module.xml

```diff
@@ -49,7 +49,7 @@
 			<makefilestub data="UMA_TREAT_WARNINGS_AS_ERRORS=1"/>
 			<makefilestub data="UMA_ENABLE_ALL_WARNINGS=1"/>
 			<makefilestub data="CFLAGS+=-U_FORTIFY_SOURCE -D_FORTIFY_SOURCE=1"/>
-			<makefilestub data="trclog.o: CFLAGS+=-U_FORTIFY_SOURCE">
```

Comment:
There must be a better way to fix this other than disabling errors for the entire file. Note this won't fix cmake builds.
Does casting to `(char *)` in the call to strncpy fix it?

Reply:
@knn-k Could you try the change in [my branch](https://github.com/keithc-ca/openj9/tree/trclog) to see if it avoids the warning?

Reply:
Thank you, @keithc-ca .  Your change works with GCC 9.3 on Ubuntu 20.04 for AArch64.

Reply:
I opened #10293.

Reply:
@pshipton I've played around with trying to fix this in omr. In general the compiler is pretty crafty and sees through any kind of casts that you do. also Re: cmake, we already handle this issue